text/0000-local-loop-bindings.md

```diff
@@ -0,0 +1,218 @@
+- Feature Name: local_loop_bindings
+- Start Date: 2018-12-25
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This will allow an extended syntax for `loop` to accept local variables, which change for each iteration to simplify more complicated loop constructs and avoid mutable state.
+
+
+# Motivation
+[motivation]: #motivation
+
+The new syntax is inspired by `loop` in the upcoming release of the [scopes programming language](scopes.rocks).
```

Comment:
scopes.rocks -> https://bitbucket.org/duangle/scopes/wiki/Home
Otherwise it got translated to the github

Might be even better mention [this](https://bitbucket.org/duangle/scopes/commits/e4025117e2e6) or [that](https://bitbucket.org/duangle/scopes/commits/47fd16cb8a28) (if I properly understood your reference)

Reply:
I'm talking about the version introduced in [this commit](https://bitbucket.org/duangle/scopes/commits/6a44e062e6a4a7813146a850c8982c0f902eefba).
It's not properly documented yet, so I thought, there is no reason in specifying it.